src/adventures-photography/_posts/2017-08-09-eurovelo-albania.md

```diff
@@ -0,0 +1,148 @@
+
+{% figure %}
+    <div class="row pair">
+        {% img src: 2016-06-web-DSCF7355, alt: Approaching Bajram Curri the previous day %}
+        {% img src: 2016-06-web-DSCF7362, alt: Approaching Bajram Curri the previous day %}
+    </div>
+    {% figcaption %}**Left:** Max verses truck; **Right:** A random butterfly that landed on the GPS during a mid-clib break. {% endfigcaption %}
+{% endfigure %}
+
+We stood there looking through the bars of these small cages at two brown bears, each separated from the other and both looking very bored. Neither of us were impressed by this, it seemed cruel to leave these animals locked up in such a small space, barely enough to stretch out; I made no effort to hide the bars when I pulled the camera out. With this we quickly hit the road again, it was now getting late in the day and we really needed to get out of this valley before dark and find somewhere we could get a proper meal and stay for the night. 
+
+{% figure narrow %}
+  {% img src: 2016-06-web-DSCF7366, alt: One of the two brown bears we found in Albania. %}
+  {% figcaption %}One of the two brown bears we found in Albania.{% endfigcaption %}
+{% endfigure %}
+
+We hit the road again with renewed energy. Eating always makes a huge difference on big days like this, and we were soon leaving the mountains and making progress towards the main roads where we hoped the terrain would level for a bit and the road surface might improve. As we reached the main road we saw it fall away in one direction and climb in the other so we checked and double checked the maps before turning right — downhill — and were soon cruising through the forests on a wide and very well-made road towards the valley floor. 
+
+{% figure %}
+    {% img src: 2016-06-web-DSCF7375, alt: Max making chase up a hill somewhere near the Fierza Reservior. %}
+    {% figcaption %}Max making chase up a hill somewhere near the Fierza Reservior.{% endfigcaption %}
+{% endfigure %}
+
+We quickly came to another post-industrial town which had a very similar feeling to Bajram Curri, so we made the call not to stop. We were aiming for the town of Puké which wasn’t too much further anyway, and acted as at the regional capital for the part of Albania we’d been cycling through all day. As we passed out the far end of the town we soon began climbing into the mountains again, except this time the climbing was over within an hour as the road levelled and we started spotting luxuriant meadows among the forrest to the right. We made a mental note of where we were as a potential camping spot before continuing. A few minutes later we arrived in Puké. 
```

Comment:
Yeah, I noticed this one too. I'm not too sure what to do about it, it seems that it's not in the font character set.